core/server/src/main/java/tachyon/master/block/BlockMasterRestServiceHandler.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.block;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import tachyon.Constants;
+import tachyon.exception.TachyonException;
+import tachyon.master.TachyonMaster;
+
+/**
+ * This class is a REST handler for block master requests.
+ */
+@Path("/")
+public final class BlockMasterRestServiceHandler {
+
+  /**
+   * Returns the service name.
+   *
+   * Method: GET
+   * Response: {@code string}
+   *
+   * @return the response object
+   */
+  @GET
```

Comment:
since I am not using all of dropwizard, this will not work out of the box ... I would prefer to investigate support for `@Timed` in a separate PR, because figuring out what are the right dependencies to include without introducing conflicts with existing dependencies is tricky